runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -125,6 +125,8 @@
 	private final EventBus eventBus;
 	private final KeyManager keyManager;
 	private Applet client;
+
+	@Getter
```

Comment:
This effectively destroys the purpose of ClientUI, I would rather not expose the frame.

Reply:
How should I not expose the frame?

Reply:
Create the notification from inside the ClientUI.